packages/dds/sequence/src/sharedString.ts

```diff
@@ -223,6 +224,7 @@ export class SharedString
 	 * @param combiningOp - Optional. Specifies how to combine values for the property, such as "incr" for increment.
 	 */
 	public annotateMarker(marker: Marker, props: PropertySet, combiningOp?: ICombiningOp) {
+		assert(props.markerId === undefined, "Cannot change the markerId of an existing marker");
```

Comment:
It should also arguably be allowed to set the property to the existing value. I don't have a strong feeling about this though.